server/couchDB/couchSeeder.js

```diff
@@ -0,0 +1,25 @@
+const createDB = require('./');
+const generateCouchData = require('./generateCouchData');
+const batchSize = 1000;
+
+(async () => {
+  const workspaces = await createDB();
+  let docs = [];
+  let batch = 0;
+  for(let i = 1; i <= 10000000; i++) {
+    const dataFormat = generateCouchData(i);
+    docs.push(dataFormat);
+    batch++;
+    if(batch === batchSize) {
+      try {
+        await workspaces.bulk({ docs });
```

Comment:
How long does this take for you? When I tried this method, it never completed. I had to write a bash script.

Reply:
It took about 45 minutes. I messed around with batch sizes, and 1000 seemed to be the best one that I landed on. It looked like your batch size was 500, which should have for sure worked. It takes a lot to hit the bulk limit size. I'd be down to go through it with you if you want.